simulation/graph.py

```diff
@@ -1,24 +1,71 @@
 import matplotlib.pyplot as plt
+#
+# def display_graphs(time, ax, ay, wheel_velocity, total_slip, yaw_rate, des_rate):
+#     ax2, ax1 = plt.subplots()
+#     ax4, ax3 = plt.subplots()
+#     ax1.plot(time, wheel_velocity, label = 'Wheel Velocity')
+#     ax1.plot(time, ax, label = 'Car Velocity')
+#     #ax1.plot(time, ay, label = 'Lateral Velocity')
+#     #ax1.set_xlim([-1, 5])
+#     ax2 = ax1.twinx()
+#     #ax2.set_ylim([0, 100])
+#     ax2.plot(time, total_slip, label = 'Slippage', color = 'Red')
+#     ax1.legend()
+#     ax2.legend()
+#     ax1.set_xlabel('Time (s)')
+#     ax1.set_ylabel('Velocity (M/s)')
+#     ax2.set_ylabel('Slippage (%)')
+#
+#     ax3.plot(time, yaw_rate, label = 'Yaw Rate')
+#     ax3.plot(time, des_rate, label = 'Des Rate')
+#     ax3.legend()
+#     ax3.set_ylabel('Yaw Rate (radians/s)')
+#     ax3.set_xlabel('Time (s)')
+#     plt.show()
```

Comment:
No need to keep these commented segments around since it's already in the git history

Reply:
Done.